www/templates/main-goals.html

```diff
@@ -73,45 +79,148 @@
+            <img src="img/ic_header_silver.png" style="height:15px; width:15px;float: left;"><p style="font-size: 10px; float: left; margin: 0 15px 0 2px;">{{silver}}</p>
+            <img src="img/ic_header_gem.png" style="height:15px; width:15px;float: left;"><p style="font-size: 10px; float: left; margin: 0 0 0 2px;">{{gem}}</p>
           </div>
           <div class="col" style="margin-left: 10%; float:left;">
             <div style="text-align:center; padding-top:6px">
-              <img src="img/ic_header_healer.png" style="height:20px; width:20px; float:left;">
+                <img in-if="profile.stats.class == 'warrior'" src="img/ic_header_warrior.png" style="height:20px; width:20px; float:left;">
+                <!--img in-if="profile.stats.class == 'healer'" src="img/ic_header_healer.png" style="height:20px; width:20px; float:left;">
+                <img in-if="profile.stats.class == 'mage'" src="img/ic_header_mage.png" style="height:20px; width:20px; float:left;">
+                <img in-if="profile.stats.class == 'rogue'" src="img/ic_header_rogue.png" style="height:20px; width:20px; float:left;"-->
               <p style="font-size:12px; float:left;padding-left:5px ">{{profile.profile.name}}</p>
             </div>
               <p style="font-size:10px; margin:0px; position:fixed; padding-top:25px">Level: {{profile.stats.lvl}}</p>
-            <button class="button button-icon" ui-sref="root.main.goals.party" style="float:right;">
+            <!--button class="button button-icon" style="float:right;" ng-click="joinParty()">
             <i class="icon ion-ios-people"></i>
-            </button>
+            </button-->
           </div>
         </a>
       </div>
-      <ion-list>
-        <ion-item class="item" ng-repeat="goal in goals" style="border:1px;">
-          <div class="row">
-          <label class="checkbox">
-              <input type="checkbox" ng-click="completeGoal(goal)">
-          </label>
-          <div>
-          <!--div class="col col-20" style="display:flex; justify-content:flex-end; align-items:center;">
-          <button class="button button-clear" style="font-size:2rem !important">
-            <i icon="ion-checkmark-circled"></i>
-          </button>
-          </div-->
-            <div class="col col-80" style="padding-left: 5% !important;">
-            <h2>{{goal.name}}</h2>
-            <p style="font-size: 10px">{{goal.note}}</p>
-            </div>
-            <ion-option-button class="button-assertive" ng-click="removeGoal(goal)">Delete
-            </ion-option-button>
+      <div class="row" id="costume-tab">
+        <div class="col">
+          <button id="challenge" class="button" ng-class="{'active': isActive}" ng-click="activeButton()" type="button">Challenges</button>
+          <button id="party" class="button" ng-class="{'active': isActiveP}" ng-click="partyBotton()" type="button">Party</button> 
```

Comment:
nit: minor typos such as `Botton`. Please do a quick spellcheck and fix in the next round.